src/pybind/mgr/nfs/ganesha_conf.py

```diff
@@ -368,6 +376,7 @@ def __init__(
         self.protocols = protocols
         self.transports = transports
         self.clients: List[Client] = clients or []
+        self.sectype = sectype
 
     @classmethod
     def from_export_block(cls, export_block: RawBlock, cluster_id: str) -> 'Export':
```

Comment:
Is there a change needed in this function as well so that the config can be loaded from a dump of the raw `export-N` rados object?

Reply:
Let me check. I haven't tested that case.

Reply:
Fixed & additional unit test cases added. Thanks for noticing this!